purchase_triple_discount/tests/test_purchase_discount.py

```diff
@@ -2,55 +2,19 @@
 # Copyright 2017 Tecnativa - David Vidal
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests import common
+from openerp.tests import common
 
 
 class TestPurchaseOrder(common.SavepointCase):
 
     @classmethod
-    def setUpClass(cls):
-        super(TestPurchaseOrder, cls).setUpClass()
-        cls.partner = cls.env['res.partner'].create({
-            'name': 'Mr. Odoo',
-        })
-        cls.product1 = cls.env['product.product'].create({
```

Comment:
Please don't remove these records by demo data, as demo data tend to change, and thus tests will fail with unexpected reasons.

Reply:
Hi.

> as demo data tend to change

sorry, don't understand.

>  thus tests will fail with unexpected reasons

don't understand neither.

Well, a good pro for having demo data, is that the end user that test the module on runbot, on test env, can more easily test the module, without having to recreate all the data demo set.

Reply:
Yes, keep both, but don't remove the creation of these records in these tests

Reply:
I understand what you ask @pedrobaeza but not why. Could you simply explain the reason or the use case that makes interesting keeping 40 lines of python code ?